tests/managed-service/test_post_installation_state.py

```diff
@@ -26,6 +27,37 @@ class TestPostInstallationState(ManageTest):
     def test_post_installation(self):
         storage_cluster.ocs_install_verification()
 
+    @acceptance
+    @ms_provider_required
+    @pytest.mark.parametrize(
+        argnames=["resource"],
+        argvalues=[
+            pytest.param(
+                *["cephblockpool"],
+                marks=pytest.mark.polarion_id("OCS-3907"),
+            ),
+            pytest.param(
+                *[constants.CEPHFILESYSTEMSUBVOLUMEGROUP],
+                marks=pytest.mark.polarion_id("OCS-3908"),
+            ),
+        ],
+    )
+    def test_consumers_connected(self, resource):
+        """
+        Test that at least one consumer is connected to the provider cluster
+        and a unique cephblockpool and subvolumegroup are successfully created
+        on the provider cluster for each connected consumer.
+        """
+        assert managedservice.get_consumer_names()
+        for consumer_name in managedservice.get_consumer_names():
```

Comment:
It would be better to store consumer names into a variable and call assert and for cycle on it. Don't call it twice as it calls oc command twice. You can also log the list of consumers so it is easier to debug.

Reply:
Thanks, done